stories/Button.stories.tsx

```diff
@@ -19,15 +21,19 @@ const meta: Meta = {
 
 export default meta;
 
-const Template: Story<ButtonProps> = (args) => <Button {...args} />;
+const Template: Story<ButtonProps> = (args) => (
+  <ChakraProvider theme={getTheme(args.currentColorMode ?? 'day')}>
```

Comment:
Do you need to do this if you're doing it in the preview.jsx file?

Reply:
Yea this is not needed anymore. Updated!